x-pack/plugins/fleet/server/routes/epm/index.ts

```diff
@@ -186,9 +192,11 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
     {
       path: EPM_API_ROUTES.INFO_PATTERN_DEPRECATED,
       validate: GetInfoRequestSchemaDeprecated,
-      fleetAuthz: {
-        integrations: { readPackageInfo: true },
-      },
+      fleetAuthz: (fleetAuthz: FleetAuthz): boolean =>
```

Comment:
didn't want to update this since it's marked deprecated but route matching will always hit this when optional `packageVersion` is omitted from path.